qa/suites/rbd/qemu/cache/writeback.yaml

```diff
@@ -1,3 +1,9 @@
+overrides:
+  ceph:
+    conf:
+      client:
+        rbd default features: 125
```

Comment:
Why is this being overridden in these tests (here and below)?

Reply:
my bad, forgot to cleanup. Removed.